diff --git a/dist/PatternVariant.js b/dist/PatternVariant.js
index 36b3f64255aee1230dd3503364b316afc8e30fee..7ab16da8871c26da61d887d8522a0fa7433e8cac 100644
--- a/dist/PatternVariant.js
+++ b/dist/PatternVariant.js
@@ -243,6 +243,10 @@ var PatternVariant = /*#__PURE__*/function () {
           for (var i = 0; i < preview.length; i += 1) {
             previewPatterns["".concat(key, "--").concat(i)] = _this2.buildPreviewPattern(preview[i]);
           }
+        } else if (field.getType() === 'pattern' && typeof preview === 'object') {
+          Object.keys(preview).forEach(i => {
+            previewPatterns["".concat(key, "--").concat(i)] = _this2.buildPreviewPattern(preview[i]);
+          })
         } else if (field.getType() === 'pattern' && preview !== null && preview !== void 0 && preview.id) {
           previewPatterns[key] = _this2.buildPreviewPattern(preview);
         }
diff --git a/dist/twigRenderEngine.js b/dist/twigRenderEngine.js
index 777539639b097f82814d99821604dff53d7196e0..05f302d5785be3e2ad3173b5d3eaf8bba2ad0ccf 100644
--- a/dist/twigRenderEngine.js
+++ b/dist/twigRenderEngine.js
@@ -245,10 +245,10 @@ function _renderPatternPreview() {
 
                     if (variant.getField(fieldName).multiValueType() === _Field.MultiValueTypes.items) {
                       if (previewRenderedVariables[nameKeys[0]] === undefined) {
-                        previewRenderedVariables[nameKeys[0]] = [];
+                        previewRenderedVariables[nameKeys[0]] = isNaN(delta) ? {} : [];
                       }
 
-                      previewRenderedVariables[nameKeys[0]][delta] = promisedPreviewValues[j];
+                      previewRenderedVariables[nameKeys[0]][isNaN(delta) ? nameKeys[1] : delta] = promisedPreviewValues[j];
                     } else if (variant.getField(fieldName).multiValueType() === _Field.MultiValueTypes.field_items) {
                       if (previewRenderedVariables[nameKeys[0]] === undefined) {
                         previewRenderedVariables[nameKeys[0]] = [];